ops/graph-deploy.js

```diff
@@ -15,7 +15,7 @@ async function deploy (opts = {}) {
   let msg
 
   /* create the subgraph */
-  if (graphNode !== 'https://api.thegraph.com/deploy/') {
+  if (graphNode !== 'https://api.thegraph.com/deploy/' && graphNode !== 'https://api.staging.thegraph.com/deploy/') {
```

Comment:
does this works ?

Reply:
The deployment itself not yet (for me) but we should have this for when it will.